app/controllers/items_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ItemsController < ApplicationController
+
+  def create
+    @user = User.find(params[:user_id])
+    @item = @user.items.build(item_params)
+    @item.user = current_user
+
+    if @item.save
```

Comment:
Sorry, meant "could" do. From a performance perspective it's the same, as both will only require a single db query. My suggestion does save a line of code, but that doesn't matter